### PR TITLE
Add slashing functionality

### PR DESCRIPTION
**Pull request by GabrielSac**
Add slashing module to the application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -526,5 +526,66 @@ Example:
...
 
+### Slashing    
+You can use the slashing CLI commands to query slashing state
+```
+alignedlayerd query slashing --help
+```
+#### Querying Slashing Params
+To query genesis parameters for the slashing module:
+```
+simd query slashing params [flags]
```
Comment (JulianGCalderon) [inline README.md:537]:
```suggestion
alignedlayerd query slashing params [flags]
```

File: multi_node_setup.sh [hunk 2]
```diff
@@ -80,6 +80,15 @@ if ! docker run --rm -it -v $(pwd)/prod-sim/$1:/root/.alignedlayer alignedlayerd
...
 
+jq '.app_state.slashing.params= {
+                        "downtime_jail_duration": "600s",
+                        "min_signed_per_window": "0.050000000000000000",
+                        "signed_blocks_window": "10000",
+                        "slash_fraction_double_sign": "0.050000000000000000",
+                        "slash_fraction_downtime": "0.000100000000000000"
+                }
+        ' prod-sim/$1/config/genesis.json|sponge prod-sim/$1/config/genesis.json
+
```
Comment (JulianGCalderon) [inline multi_node_setup.sh:91]: All nodes should have slashing config

**Review by JulianGCalderon — CHANGES_REQUESTED** [review]

**Review by JulianGCalderon — CHANGES_REQUESTED** [review]
Add "sponge" to requirements of the docker setup.

**Review by MauroToscano — APPROVED** [review]

**Review by JulianGCalderon — APPROVED** [review]
LGTM

**Pull request merged**